src/pages/ChatPage.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import './ChatPage.css';
+
+const ChatPage = ({location, history}) => {
+    const name = location.state;
+    const [message, setMessage] = useState([]);
+    const userText=['안녕하세요', '샘플 데이터입니다', '반가워요', '날씨가 어때요?', '감사합니다', '좋은하루 되세요', '사랑해요ㅎ.ㅎ'];
```

Comment:
```suggestion
    const userTexts=['안녕하세요', '샘플 데이터입니다', '반가워요', '날씨가 어때요?', '감사합니다', '좋은하루 되세요', '사랑해요ㅎ.ㅎ'];
```
OR
```suggestion    
    const userTextList=['안녕하세요', '샘플 데이터입니다', '반가워요', '날씨가 어때요?', '감사합니다', '좋은하루 되세요', '사랑해요ㅎ.ㅎ'];
```
userText 라고 변수명을 지으면 느낌이 String 변수의 느낌이 나네요.
실상은 List 인데 말이죠
보통 복수형, List 등을 붙입니다.